english/eating-walls/eating-marshmallows.md

```diff
@@ -28,6 +28,8 @@ Do you:
 
 [Eat ALL the marshmallows!](../count-the-marshmellows/eat-all-the-marshmellows/eat-all-the-marshmellows.md)
 
+[Eat pani-puri](../eat-panipuri/eat-panipuri.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.